src/VoyagerServiceProvider.php

```diff
@@ -97,6 +97,7 @@ public function boot(Router $router, Dispatcher $event)
 
         if (app()->version() >= 5.4) {
             $router->aliasMiddleware('admin.user', VoyagerAdminMiddleware::class);
+            $this->loadMigrationsFrom(realpath(__DIR__.'/../publishable/database/migrations'));
```

Comment:
This may cause errors due to the migrations running twice if someone has already published their migrations.  Also, there's no need to keep it inside the 5.4+ check